This patch reverts Qremote to qmail-remote behaviour if the message contains
8 bit characters and the remote host does not announce 8BITMIME: just send
the mail. This behaviour has caused no problems to me (or at least I never
noticed them) but it violates the RfC. You might need this patch if some of
your clients send mails with "Content-Transfer-Encoding: 8bit". Hopefully
I will add quoted-printable recoding one day and this can go away then.

Index: qrdata.c
===================================================================
--- qrdata.c	(revision 580)
+++ qrdata.c	(working copy)
@@ -408,8 +408,12 @@
 	
 	if (multipart) {
 #warning FIXME: add proper quoted-printable recoding here
+#if 0
 		write(1, "Z4.6.3 message has 8 Bit characters but next server does not accept 8BITMIME", 77);
 		exit(0);
+#else
+		send_plain(msgdata + off, msgsize - off);
+#endif
 	} else {
 		recode_qp(msgdata + off, msgsize - off);
 	}
